src/content/docs/dropins/cart/containers/cart.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Cart container
```

Comment:
We will no longer have a Cart container, once [USF-1610](https://jira.corp.adobe.com/browse/USF-1610) is completed. We can remove this reference.

Reply:
I will remove the Cart container docs upon completion of that ticket. Note that we might need replacement documentation.